app/domain/etl/item/content/content_populator.rb

```diff
@@ -0,0 +1,11 @@
+class Etl::Item::Content::ContentPopulator
+  def self.process
+    new.process
+  end
+
+  def process
+    Dimensions::Item.where(document_text: nil).each do |item|
```

Comment:
It might be better to rely on [find_each](http://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each) so we don't overload the process memory?